src/types/lottoType.ts

```diff
@@ -0,0 +1,20 @@
+export interface WinningNumber {
+  winningNumbers: number[];
```

Comment:
p2: 요런 느낌도 괜찮을 것 같네요.
```suggestion
  lottoNumbers: number[];
```